docs/stellar_satellite_open-stream.md

```diff
@@ -16,7 +16,7 @@ stellar satellite open-stream [satellite-id] [flags]
 ### Options
 
 ```
-      --accepted-framing strings    Framing type to receive. One of: AX25|IQ|IMAGE_PNG|IMAGE_JPEG|FREE_TEXT_UTF8|WATERFALL|BITSTREAM
+      --accepted-framing strings    Framing type to receive. One of: BITSTREAM|AX25|IQ|IMAGE_PNG|IMAGE_JPEG|FREE_TEXT_UTF8|WATERFALL
```

Comment:
I can't understand why the order of this keep getting pushed around when regenerating docs.

Reply:
It is slightly annoying :)